CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # CHANGELOG
 
+## v0.56.0
+
+*November 15, 2024*
+
+This release upgrades the `ibc-rs` and `tendermint-rs` dependencies to `v0.56.0`
+and `v0.40.0`, respectively.
+
```

Comment:
I am curious why the breaking changes entry is not here.

Reply:
Oops! [d4d7c42](https://github.com/informalsystems/cosmwasm-ibc/pull/6/commits/d4d7c421fd47f44807a389100b2255806266c406)